src/intrinsics/ecma262/Error.js

```diff
@@ -74,10 +74,10 @@ function buildStack(realm: Realm, context: ObjectValue) {
     msg.throwIfNotConcrete();
   }
 
-  for (let executionContext of stack) {
+  for (let executionContext of stack.reverse()) {
     let caller = executionContext.caller;
     let locString = describeLocation(realm, caller ? caller.function : undefined, caller ? caller.lexicalEnvironment : undefined, executionContext.loc);
-    if (locString !== undefined) lines.unshift(locString);
+    if (locString !== undefined) lines.push(locString);
```

Comment:
"Known" might be a strong word, since I'm very new to JS :)
I'm assuming that arrays behave more or less like Arrays/Vectors in other languages. If that's the case, then calling ```unshift``` n times will result in O(n^2), because for every call we are copying all the previously inserted items, where's pushing would be O(n), with reverse taking either O(n) (if it actually creates a new copy) or O(1) (if it works like an iterator adapter, though my guess was that in JS it's the former anyway), with a total complexity of O(n).

Are my assumptions regarding JS wrong here or anything else I'm missing?